misaligned.cpp

```diff
@@ -1,21 +1,50 @@
 #include <assert.h>
 #include <iostream>
+#include <sstream>
+#include <string>
 
-int printColorMap() {
+int printColorMap(std::ostream& out) {

```

Comment:
Good choice to accept the output stream. This can print to cout, or it can print to a string as well.